lib/utils/validate-addons.js

```diff
@@ -1,30 +1,28 @@
 const exists = require('./npm-exists');
 const resolvePackages = require('./resolve-packages');
 
+module.exports = (addons) => addons.map( pkg => checkExistence(pkg));
```

Comment:
1. Let's always use `function` declarations when defining functions
1. Let's name this function `validateAddons`
1. There is no need to create the closure in map / create a separate function
1. Please add JSDoc to the exported function since this is the API

```javascript
module.exports = function validateAddons(addons) {
  return addons.map(checkExistence);
}
```